mautic/mautic
# The form number field can be set to decimal precision

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | [ ]
| New feature/enhancement? (use the a.x branch)      | [ ]
| Deprecations?                          | [ ]
| BC breaks? (use the c.x branch)        | [ ]
| Automated tests included?              | [ ] 
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | Fixes #... 



#### Description:
 This PR adds the option to set the decimal precision for a form number field.

![image](https://github.com/mautic/mautic/assets/96085911/fec8d970-7bde-4609-861f-f7cfef271307)



#### Steps to test this PR:


1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. Create a new form.
3. Add a new field with the number type.
4. Set a precision on the property tab.
5. Mapped the field with a contact field.
6. Submit a form. 
7. Check how it works.
